app/components/money-display.tsx

```diff
@@ -1,18 +1,7 @@
-import type { ReactNode } from 'react';
-import type { Currency, CurrencyUnit, Money } from '~/lib/money';
+import type { Currency, CurrencyUnit } from '~/lib/money';
+import { Money } from '~/lib/money';
 import { cn } from '~/lib/utils';
 
-type MoneyDisplayBaseProps = {
-  /** The symbol to display */
-  symbol: string;
-  /** Place the symbol before or after the number */
-  symbolPosition: 'prefix' | 'suffix';
-  /** The number to display */
-  children: ReactNode;
-  size?: 'sm' | 'default';
-  className?: string;
-};
-
 const sizes = {
```

Comment:
we can probably even get rid of sizes if this component will be only used for displaying input values and we don't plan to have different sizes for that